tests/cypress/plugins/imageGenerator/addPlugin.js

```diff
@@ -1,33 +1,52 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2020-2021 Intel Corporation
 //
 // SPDX-License-Identifier: MIT
 
+// eslint-disable-next-line no-use-before-define
 exports.imageGenerator = imageGenerator;
 
-const jimp = require('jimp');
 const path = require('path');
+const jimp = require('jimp');
 
-function imageGenerator(args) {
-    const directory = args.directory;
-    const fileName = args.fileName;
-    const width = args.width;
-    const height = args.height;
-    const color = args.color;
-    const posX = args.posX;
-    const posY = args.posY;
-    const message = args.message;
-    const file = path.join(directory, fileName);
-    const count = args.count;
+function createImage(width, height, color) {
+    return new Promise((resolve, reject) => {
+        // eslint-disable-next-line new-cap, no-new
+        new jimp(width, height, color, ((err, img) => {
+            if (err) reject(err);
+            resolve(img);
+        }));
+    });
+}
+
+function saveImage(image, posX, posY, message, file, index, extension) {
     return new Promise((resolve, reject) => {
+        jimp.loadFont(jimp.FONT_SANS_64_BLACK, (err, font) => {
+            if (err) reject(err);
+            image.print(font, Number(posX), Number(posY), `${message}. Num ${index}`)
```

Comment:
Adding text to an image should not be a part of ``createImage()``? Or maybe ``appendText()``. I don't think that ``saveImage()`` is a good place

Reply:
Done. Changed this function into `appendText` and removed save from here.